docs/source/markdown/podman-images.1.md

```diff
@@ -1,72 +1,67 @@
 % podman-images(1)
 
 ## NAME
+
 podman\-images - List images in local storage
 
 ## SYNOPSIS
+
 **podman images** [*options*]
 
 **podman image list** [*options*]
 
 **podman image ls** [*options*]
 
 ## DESCRIPTION
-Displays locally stored images, their names, and their IDs.
+
+**podman images** displays locally stored images, their names, and their IDs.
 
 ## OPTIONS
 
 #### **--all**, **-a**
 
-Show all images (by default filter out the intermediate image layers). The default is false.
-
-#### **--digests**
-
-Show image digests
+Show all images. If set, itermediate layers are not filtered out.
 
-#### **--filter**=*filter*, **-f**
+**Note:**
 
-Filter output based on conditions provided
+- Intermediate layers are filtered out by default i.e. when this option is not set.
 
-  Filters:
-
-  **since=IMAGE**
-    Filter on images created after the given IMAGE (name or tag).
-
-  **before=IMAGE**
-    Filter on images created before the given IMAGE (name or tag).
+#### **--digests**
 
-  **dangling=true|false**
-    Show dangling images. Dangling images are a file system layer that was used in a previous build of an image and is no longer referenced by any active images. They are denoted with the `<none>` tag, consume disk space and serve no active purpose.
+Show image digests.
 
-  **label**
-    Filter by images labels key and/or value.
+#### **--filter**, **-f=*filter***
```

Comment:
The asterisks are misplaced.
```suggestion
#### **--filter**, **-f=**filter**
```

Reply:
Should be `#### **--filter**=, **-f**=**filter**`